addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2052,8 +2053,8 @@ var FieldBinaryImage = AbstractFieldBinary.extend({
                 this.$el.addClass(this.attrs.class);
             }
 
-            const image_field = this.field.manual ? this.name:'image_128';
-            var urlThumb = this._getImageUrl(this.model, this.res_id, image_field, unique);
+            const image_field = this.field.manual ? this.name : 'image_128';
+            const urlThumb = this.value ? this._getImageUrl(this.model, this.res_id, image_field, unique) : this.placeholder;
```

Comment:
What is this change fixing?
As this is in framework it would be nice to have a test covering it.

Reply:
There was one view where no value was set and we did a request to web/image, therefore showing the camera placeholder instead of the widget set placeholder

Reply:
Avoid using placeholder there?